### PR TITLE
Abe added answers to cp-o1

### PR DESCRIPTION
**Pull request by aeaghannam**
First few questions were easy however the last four were a bit challenging.

### CODE REVIEW

**Comment by aeaghannam** [comment]
First few questions were easy however the last four were a bit challenging.

File: README.md [hunk 1]
```diff
@@ -47,15 +47,20 @@ Set the margin to 0 for each direct child of:
 - `.main`
 - `aside`
 
-> Commit
+--- a css code was added that changes the margin of the direct child tags for the elements listed above:
+""}
+header, footer, aside, .main > span{
+  margin: 0;
+} ""
+
```
Comment (jsm13) [inline README.md]: the `>` (child relation selector) is not transitive so need to use it for each. `header>*, footer>*, aside>*, .main>*`

File: README.md [hunk 2]
```diff
@@ -71,7 +76,7 @@ Instead of making `.main` red, reorder the elements to...
 
 Do this using only CSS -- do not rearrange the HTML.
 
-> Commit
+--- In the css "media element" the red effect was removed and a width and margin setting was added so all the html elements will populate in order.
```
Comment (jsm13) [inline README.md]: looking for flexbox here

File: README.md [hunk 3]
```diff
@@ -82,4 +87,9 @@ Rearrange the lines to identify the correct workflow for submitting a pull reque
 
 Remove the lines that are not required in this workflow.
 
-> Commit
+--- the following steps were re-ordered and remained:
+fork on github
+git clone
+git add .
+git commit -m ""
+git push origin my-solution
```
Comment (jsm13) [inline README.md]: 👍

**Pull request closed without merging**